migrations/20241230195405_add_case_insensitive_username_indexes.sql

```diff
@@ -0,0 +1,4 @@
+-- Add migration script here
+CREATE UNIQUE INDEX names_username_lower_idx ON names (LOWER(username));
+
+CREATE UNIQUE INDEX old_names_username_lower_idx ON old_names (LOWER(old_username));
```

Comment:
this will fail if we already have non-unique usernames wont it?

Reply:
ah good catch let me see

Reply:
Will wait to merge this till we run the dedup script on the usernames table

Reply:
actually I just removed the unique constraint. This is fine there wont be too many duplicates anyways